### ISSUE TITLE
[Feature request] Pomotroid for Android / iOS

### ISSUE THREAD

**Issue by SuriyaaKudoIsc** [Splode/pomotroid#52]
Pomotroid for Android and iOS would be great!

**Comment by PatrickLohan** [comment]
May I give this a go @Splode ? I'd be using {N}ativescript-vue. I guess I'd create a feature branch?

**Comment by Splode** [comment]
@PatrickLohan Yes, please do! Feature branch with a PR to `dev` would be my suggestion.

Let me know if there's any way I can help.

**Comment by omaralvarez** [comment]
This would be awesome.

**Comment by jhorsman** [comment]
Pomotroid on mobile device would be great!

**Comment by Draketheb4dass** [comment]
Hello guys, been using the mac version for over a year now. I want to build one for Android using Kotlin. Will link the github repo when ready here.
